frontend2/src/views/Tournaments.tsx

```diff
@@ -33,12 +34,14 @@ const Tournaments: React.FC = () => {
     },
     queryClient,
   );
+
   const currentTournamentText = tournamentsText[episodeId];
-  const hasContent = Object.values(currentTournamentText).some(
-    (value) => value !== "",
-  );
+  const hasContent =
+    !isNil(currentTournamentText) &&
+    Object.values(currentTournamentText).some((value) => value !== "");
+
   if (!hasContent) {
-    return <NoContentLegacyEpisode />;
+    return <NoContentFound />;
```

Comment:
so to display the tournaments that are registered in a released episode, this code now requires that content is specified for the tournaments page. i think this is fine.